start

```diff
@@ -20,6 +20,10 @@ show_help() {
 
 while :; do
   case $1 in    
+    -D|--data-only)
```

Comment:
Why uppercase -D instead of lowercase to match -h?

Reply:
It was uppercase in the pay-core repo. 

Probably less confusing to keep it lowercase